### ISSUE TITLE
package doesn't install

### ISSUE THREAD

**Issue by Vulonus** [benkehoe/aws-sso-util#104]
I'm having problems installing the package with `pipx`.
I started troubleshooting, since the issue is building `pyyaml`.
```
pip seemed to fail to build package:
    pyyaml<6.0.0,>=5.3.1

Some possibly relevant errors from pip install:
    error: subprocess-exited-with-error
    AttributeError: cython_sources
Error installing aws-sso-util.
```
As stated in the issue linked below, this seems to be a problem since some Cython release around 3.0.0
#https://github.com/yaml/pyyaml/issues/601

If anyone has a hotfix for this, please let me know, I'm pressed for time...

**Comment by thetoolsmith** [comment]
Same for me.

**Comment by benkehoe** [comment]
Updated to PyYAML 6.0.1, shipped as v4.32 (no aws-sso-lib changes). Please let me know if you continue to have trouble, or experience any problem with the CloudFormation support (which is where PyYAML is used).

**Comment by benkehoe** [comment]
Due to no further complaints, I'm closing this as fixed.
